lib/computer.ex

```diff
@@ -4,6 +4,65 @@ defmodule Computer do
   def move(board, space) do
     Board.make_mark(board, space, %Computer{}.mark)
   end
-  
-  
+
+  def get_best_move(game, depth \\ 0, best_score \\ []) do
```

Comment:
Is it possible to change the name of `best_score`?  `list_of_scores`?  `scores`?